.github/workflows/increment-version-on-merge.yml

```diff
@@ -45,6 +45,15 @@ jobs:
       - name: Increment the version
         uses: im-open/git-version-lite@main
         with:
-          create-ref: true
+          create-ref: false
           github-token: ${{ secrets.GITHUB_TOKEN }}
           default-release-type: major
+          
+      - name: Create Version and Major Tags
```

Comment:
I can't really test this without merging.  Any ideas on how I could test this out prior?  @mike-schenk

Reply:
I haven't tried to do this myself, but it seems like you should be able to fork or otherwise copy this repository and merge into _that repository's_ main branch.

Reply:
I don't have access to the settings of this repo.  Is the user `github-actions` added as an admin that has permissions to the git repo?

Reply:
I noticed this is the user being used in the build workflow to commit changes.  I assume the same user could be used to push the tags.

Reply:
I also am assuming in this repo that the "Read and write permissions" is enabled.
![image](https://user-images.githubusercontent.com/8923247/222829489-3efe56aa-2d03-4298-83d8-76f4b16e8214.png)

Reply:
Asserted: https://github.com/bradyclifford/git-version-lite/actions/runs/4327343631